src/python/grpcio_tests/tests/testing/_server_application.py

```diff
@@ -25,9 +27,25 @@
 class FirstServiceServicer(services_pb2_grpc.FirstServiceServicer):
     """Services RPCs."""
 
+    def __init__(self):
+        self._abort_lock = threading.RLock()
+        self._abort_response = _application_common.ABORT_NO_STATUS_RESPONSE
+
     def UnUn(self, request, context):
         if _application_common.UNARY_UNARY_REQUEST == request:
             return _application_common.UNARY_UNARY_RESPONSE
+        elif request == _application_common.ABORT_REQUEST:
```

Comment:
optional: can we order the components of comparison to be "expected" == "actual" for all three of them, or the other way around?